pom.xml

```diff
@@ -290,6 +290,11 @@
       <version>${bdg-formats.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.bdgenomics.qc-metrics</groupId>
+      <artifactId>qc-metrics-core</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```

Comment:
See [`Concordance.scala`][1], which `GermlineStandardCaller` depends on.

I'm happy to rip out all that code instead if it's unused.

[1]: https://github.com/hammerlab/guacamole/blob/cf8ce251a023571e7fa38f01838ec973953c00f5/src/main/scala/org/hammerlab/guacamole/Concordance.scala#L27